src/components/common/header/AuthenticatedHeader.js

```diff
@@ -38,18 +45,36 @@ export default () => {
           </ul>
         </div>
       </nav>
-
       <ul className="sidenav" id="mobile-demo">
         <li><Link to="*"><i className="material-icons">post_add</i>New article</Link></li>
         <li><Link to="*"><i className="material-icons">account_circle</i>Profile</Link></li>
-        <li><a href="#!"><i className="material-icons">exit_to_app</i>Sign out</a></li>
+        <li><a href="#*" onClick={logoutUser}><i className="material-icons">exit_to_app</i>Sign out</a></li>
```

Comment:
Can `a` be changed to a `Link` too, like on line 26?

Reply:
Okay. I'll do that

Reply:
done that.